src/data_collection/models.py

```diff
@@ -9,4 +11,4 @@ class Data(models.Model):
 	log = JSONField()
 
 	def __str__(self):
-		return '%s: %s' % (self.data_type, self.log)
\ No newline at end of file
+		return '%s: %s %s' % (self.data_type, self.log, datetime.now(pytz.timezone('US/Pacific')))
```

Comment:
i meant `date_created`

Reply:
Yeah I figured that out later.. Changed it